certif/app/components/certification-candidate-in-staging-item.hbs

```diff
@@ -30,6 +30,7 @@
           @options={{hash inputFormat='dd/mm/yyyy' outputFormat='yyyy-mm-dd'}}
           @class="certification-candidates-input {{if this.birthdateFocused 'focused'}}"
           {{on 'focus' this.focusBirthdate}}
+          @value={{undefined}}
```

Comment:
**thought:** je me demande si on ne pouvait pas mieux utiliser la dépendance ember-inputmask et notamment exploiter correctement le double binding sur l'attribut' `candidate.birthDate` avec les propriétés `value` ou `unmaskedValue`. 

Dans ce cas, pourquoi ne pas mettre directement `this.args.candidateData.birthDate` ?

Reply:
Ah ! C'est ce que j'avais mis mais @aceol m'a fait douter. En fait dans tous les cas comme ce qu'on lui passe est inutile du à la fonction en dessous qui permet d'update la valeur, je pense que mettre le `undefined` permet de faciliter cette compréhension.